frontend/src/app/header-bar/header-bar.component.html

```diff
@@ -19,37 +19,12 @@ <h1>
             Charts
           </a>
         </li>
-        <li *ngIf="!loggedIn" class="separator"></li>
-        <li *ngIf="!loggedIn">
-          <a class="reverse" href="{{config.backendHostname}}/auth">
```

Comment:
And what about this?

Is this patch doing more than what the description says?

Reply:
I updated the description to mention removal of the OAuth login. The chartsvc API doesn't support it, we may want to replace it with the LinuxFoundation identity login further down the line, but right now Monocular does not need it.

Reply:
After some thought, I've decided to keep the auth code in here, even though it is currently unused. This will make it easier for us to add LinuxFoundation login in the future.